src/utilities.h

```diff
@@ -29,5 +29,6 @@ bool Str2Bool(const CStdString str);
 
 bool EndsWith(CStdString const &fullString, CStdString const &ending);
 bool StartsWith(CStdString const &fullString, CStdString const &starting);
+CStdString GetDirectoryPath(CStdString const &path);
```

Comment:
I wrote this function for an earlier fix of the advocate99 bug.  I ended up
doing the fix differently and didn't need this function anymore, but as a
utility function I thought it worth keeping, thinking we might want it in
the future.  I can remove it if you want.

 

  _____  

From: Ryan Gribble [mailto:notifications@github.com] 
Sent: Thursday, March 03, 2016 1:15 AM
To: kodi-pvr/pvr.wmc
Cc: krustyreturns
Subject: Re: [pvr.wmc] add buffer timing support and a bug fix (#39)

 

In src/utilities.h
<https://github.com/kodi-pvr/pvr.wmc/pull/39#discussion_r54853014> :

> @@ -29,5 +29,6 @@ bool Str2Bool(const CStdString str);
>  
>  bool EndsWith(CStdString const &fullString, CStdString const &ending);
>  bool StartsWith(CStdString const &fullString, CStdString const
&starting);
> +CStdString GetDirectoryPath(CStdString const &path);

This function doesnt seem to be being called by anything?

-
Reply to this email directly or view it on
<https://github.com/kodi-pvr/pvr.wmc/pull/39/files#r54853014>  GitHub.
<https://github.com/notifications/beacon/AFuXcLo3PBC4rfBmgjpTFDWDpbitVb39ks5
ppqargaJpZM4Hm_6P.gif>